Bottom: b7be02c28b0a823e4febc6d0fe6280aca0572320
Top:    259986182784ba11339361aab4f857492f51a29d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:50:19 +0100

draw.c: Remove API ClearAndRedrawOutput(), in favour of Redraw().

These do the same thing anyway.


---

diff --git a/src/draw.h b/src/draw.h
index c60b6d9..7f4d3d0 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -37,7 +37,6 @@
 
 void Draw (void);
 void Redraw (void);
-void ClearAndRedrawOutput (void);
 void DrawVia (PinTypePtr);
 void DrawRat (RatTypePtr);
 void DrawViaName (PinTypePtr);
